contracts/cw-storage/src/error.rs

```diff
@@ -11,6 +11,9 @@ pub enum ContractError {
 
     #[error("{0}")]
     Bucket(#[from] BucketError),
+
+    #[error("Object is pinned")]
+    Pinned {},
```

Comment:
I'd prefer (to be in line with `BucketError:ObjectAlreadyStored`).

```suggestion
    #[error("Object is already pinned")]
    ObjectAlreadyPinned {},
```